src/sdk_container

```diff
@@ -24,13 +24,14 @@ function run_in_sniper() {
 
         image="registry.gitlab.steamos.cloud/steamrt/sniper/sdk:latest"
         echo "Running script inside $image"
+        mkdir ~/.ccache #create directory because it might not exists which causes the mount to fail
```

Comment:
I'd use `-p` so it doesn't fail when it already exists
```suggestion
        mkdir -p ~/.ccache #create directory because it might not exists which causes the mount to fail
```

But really, you want it to match what's actually mounted so:
```suggestion
        mkdir -p "${CCACHE_DIR:-$HOME/.ccache}"
```